### PR TITLE
Add more IA tag support

### PR DESCRIPTION
**Pull request by MichaelButkovic**
@mparent61 @spra85 @benghaziboy @collin

### CODE REVIEW

**Comment by MichaelButkovic** [comment]
@spra85 adding in subhead stuff now

**Comment by benghaziboy** [comment]
👍

File: bulbs/instant_articles/parser.py [hunk 1]
```diff
@@ -74,7 +74,12 @@ def parse_instagram(tag):
 
 
 def parse_text(tag):
-    if tag.name == 'p':
+    if (tag.name == 'p' or
```
Comment (mparent61) [inline bulbs/instant_articles/parser.py]: Easier: ```tag.name in ['blockquote', 'ol', ....]```

**Comment by mparent61** [comment]
LGTM

File: bulbs/instant_articles/parser.py (hunk 1 above)
Comment (MichaelButkovic) [inline bulbs/instant_articles/parser.py]: good point. updated

**Comment by spra85** [comment]
👍

**Pull request merged**